clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/callbacks/Utils.groovy

```diff
@@ -137,6 +137,54 @@ class Utils {
     }
   }
 
+  static String getZoneFromGroupUrl(String fullUrl) {
+    if (!fullUrl) {
+      return fullUrl
+    }
+
+    def urlParts = fullUrl.split("/")
+
+    if (urlParts.length < 4) {
+      throw new IllegalFormatException("Server group Url ${fullUrl} malformed.")
```

Comment:
The casing of `Url` is inconsistent in this method. I'd go with `url` throughout.